### ISSUE TITLE
[Fixed in GH-6052] Improve Visual Focus on StoreFinder

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#4732]
This "Back to List" button & 'Get Directions' button look like has 2 outlines, one blue and one grey.
![Screen Shot 2019-09-25 at 8 53 33 AM](https://user-images.githubusercontent.com/45409515/65602585-1ef2b880-df72-11e9-8e7b-1e8727dd0b5f.png)

**Comment by KateChuen** [comment]
Tested on dev3 on chrome, safari & firefox.
Still missing some visual focus. #6589 
And also same issue with pagination as on PLP: #6568
